slides.typ

```diff
@@ -94,74 +178,33 @@
 #let slides(
     title: none,
     author: none,
+    subtitle: none,
     short-title: none,
     short-author: none,
     date: none,
-    color: teal,
-    document_body
+    theme: slides-default-theme(),
+    body
 ) = {
-    show heading.where(level: 1): h => {
-        section.update(h.body)
-    }
-
-    show heading.where(level: 2): h => {
-        pagebreak(weak: true)
-        logical-slide.step()
-        h
-    }
-
     set text(
         size: 25pt,
     )
```

Comment:
This should be configurable.

Reply:
True... Also the font, which so far needs to be set extra by the user. Is that a property of the theme or of the presentation as a whole, though?

So should the user write
```typ
#show: slides.with(
    theme: theme-name(font-size: 42pt)
)
```
or
```typ
#show: slides.with(
    theme: theme-name(...),
    font-size: 42pt,
    // or possibly
    general-typography: (
        font-size: 42pt,
    )
)
```

Reply:
Well, I guess it can be sorta both - allow the theme to decide it but also let the user override it if they want